docs/News.md

```diff
@@ -1,3 +1,10 @@
+## June 12, 2019
+ *   **Added data**  of 1350 samples from 3 studies:
+     *  [Pheochromocytoma and Paraganglioma (TCGA, Cell 2017)](https://www.cbioportal.org/study?id=pcpg_tcga_pub) *178 samples*
```

Comment:
When do you decide to add a year to the stable study id?

Reply:
We usually do but for this case, we followed the general TCGA pub id.